projects/js-packages/number-formatters/package.json

```
@@ -0,0 +1,38 @@
+{
+	"name": "@automattic/jetpack-number-formatters",
+	"version": "0.1.0",
+	"description": "Number formatting utilities",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/js-packages/number-formatters/#readme",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/js-packages/number-formatters"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Number Formatters"
+	},
+	"dependencies": {
+		"@babel/runtime": "^7"
+	},
+	"devDependencies": {
+		"@babel/core": "7.26.0",
+		"@babel/preset-react": "7.26.3",
+		"@jest/globals": "29.4.3",
+		"@types/jest": "29.5.14",
+		"jest": "29.7.0",
+		"jest-environment-jsdom": "29.7.0",
+		"typescript": "5.0.4"
+	},
+	"type": "module",
+	"exports": {
+		".": "./index.ts"
```

This is one of the things I was referring to in https://github.com/Automattic/jetpack/pull/42317#issuecomment-2730139006 about setting TypeScript up correctly. The package should offer JS modules and `.d.ts` types rather than the `.ts` file correctly.

To do that, you'd do like this here:
```suggestion
		".": {
			"jetpack:src": "./index.ts",
			"types": "./build/index.d.ts",
			"default": "./build/index.js"
		}
```
(and then set things up elsewhere to produce those `./build/` files)